.github/workflows/test.yml

```diff
@@ -101,7 +101,29 @@ jobs:
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
       - run: make slim-builds
-      - run: make test-components
+      - run: make test
```

Comment:
Does this cause us to re-run some tests?

Reply:
I'm not sure I follow the question fully. Re-run in other jobs or in this job?

Reply:
Never mind, I was missing seeing something in the `Makefile`.

However, this raises the reverse question: Does anything run `make test-components` after this change? AFAICT both `make test-components` and `make test-benches` run the `test` target, but not vice versa.

Reply:
You know, I had thought there was something but I believe you're correct: it is an orphan after this.

Reply:
24d7ce0 removes this target.